### ISSUE TITLE
[release/9.0.1xx] Build break:  error NU1903: Warning As Error: Package has a known high severity vulnerability

### ISSUE THREAD

**Issue by MichaelSimons** [dotnet/source-build#4660]
This is affecting several legs in this [failing build](https://dev.azure.com/dnceng/internal/_build/results?buildId=2556294) (Internal Microsoft link)

```
    /vmr/src/arcade/src/Microsoft.DotNet.NuGetRepack/tasks/Microsoft.DotNet.NuGetRepack.Tasks.csproj : error NU1903: Warning As Error: Package 'System.IO.Packaging' 9.0.0-rc.1.24431.7 has a known high severity vulnerability, https://github.com/advisories/GHSA-f32c-w444-8ppv [/vmr/src/arcade/Arcade.sln]

##[error]/vmr/src/arcade/src/Microsoft.DotNet.NuGetRepack/tasks/Microsoft.DotNet.NuGetRepack.Tasks.csproj(0,0): error NU1903: (NETCORE_ENGINEERING_TELEMETRY=Restore) Warning As Error: Package 'System.IO.Packaging' 9.0.0-rc.1.24431.7 has a known high severity vulnerability, https://github.com/advisories/GHSA-f32c-w444-8ppv
```

**Comment by MichaelSimons** [comment]
This is blocking PRs as well.  Example - https://dev.azure.com/dnceng-public/public/_build/results?buildId=835442&view=logs&j=609589e2-4f74-5576-cdb7-914bcaea778b&t=5ba03121-3cea-598c-07fa-53815ace41ff

**Comment by MichaelSimons** [comment]
Does this need https://github.com/dotnet/sdk/pull/43993 to fix?

**Comment by MichaelSimons** [comment]
Perhaps we need to globally no-warn NU1903 to unblock.  The fsharp fix needed to unblock https://github.com/dotnet/sdk/pull/43993 won't pass PR validation because of this issue.

**Comment by MichaelSimons** [comment]
This was fixed by dotnet/sdk#44028.
